scripts/test_memory_spikes.py

```diff
@@ -13,7 +13,7 @@
 
```

Comment:
The comments in lines 8, 98, and 141 still refer to _random inserts_. It might be good to update them and add a more general description of the workload.